utils/sn_flux_est.py

```diff
@@ -229,7 +229,7 @@ def analyse_pulse_prof(prof_path=None, prof_data=None, period=None, verbose=Fals
     return [sn, u_sn, flags, w_equiv_bins, u_w_equiv_bins, w_equiv_ms, u_w_equiv_ms, scattered]
 
 #---------------------------------------------------------------
-def pulsar_beam_coverage(obsid, pulsar, beg=None, end=None, ondisk=False):
+def pulsar_beam_coverage(obsid, pulsar, beg=None, end=None, ondisk=False, pow=0.3):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'pow'](https://app.codacy.com/manual/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4856257)